src/utils/translate/get_diff.py

```diff
@@ -0,0 +1,40 @@
+import json
+
+en = json.load(open("en.json", "r", encoding="utf-8"))
```

Comment:
let's just turn this into a function that accepts to file paths and returns the missing keys. much easier to reuse and run. to be honest, let's also make cleanup translate a function too and put these two functions in a python file called translate_utils and then we just have two functions that we can call when we need. that way you can call get_diff with old and new german file paths and you don't have stuff hardcoded so it's more extendable. does that make sense? then we just call these functions on the fly when we want to update de.json or clean up the keys in these files